.editorconfig

```diff
@@ -281,6 +278,9 @@ dotnet_diagnostic.CA1720.severity = none
 # Naming conflict
 dotnet_diagnostic.CA1724.severity = none
 
+# Member is explicitly initialized to its default value
+dotnet_diagnostic.CA1805.severity = none
+
```

Comment:
These new rules are coming in CodeAnalyzer version `3.3.0`, which I'm running locally. Figured I might as well add the exceptions here

Reply:
Fine by me. CA1805 is one I violate way more than I should.

Reply:
I actually like some instances of `CA1805`. For example, the rule currently complains when you initialize a `bool` to `false`, since `false` is the default value. However, in my opinion, initializing a bool is essential for clarity. (This is, of course, coming from my C/C++ background where you need to explicitly initialize everything or you'll be using garbage memory)

We'll cross that bridge when we get there 😄

Reply:
I do agree there. It's another one where I wish there could be a more contextual nature to the rule. I like initializing bools to false even when I know that's what it is by default when they're a field, but if it's an auto property, I like to leave the initialization off as it's ugly.

Reply:
The `eslint` ecosystem is really nice because you tend to be able to fine-tune rules like that. We'll figure out how we want to work with/around the dotnet ecosystem limitations